### PR TITLE
Expose some of AdminApp guards

### PR DESCRIPTION
**Pull request by Novkirishki**

### CODE REVIEW

**Review by jmeleshko — APPROVED** [review]

File: tests/e2e/app-specs/verify-extensions.e2e-spec.ts [hunk 1]
```diff
@@ -56,6 +62,25 @@ const ENTITY_MAP = new Map<string, any>()
...
+    it("verify config guard", async () => {
+        await BrowserNavigate(PRINT_PREVIEW_URL);
+        const currentUrl = await BrowserGetUrl();
+        expect(currentUrl).toBe(CONFIG_PAGE_URL);
+    });
+
+    it("verify auth guard", async () => {
+        await BrowserNavigate(BASE_URL);
+        const setLocalStorageString = `localStorage.setItem("sf.config.serviceUrl","${URL_IN_CONFIG_FILE}");`;
+        await browser.executeScript(setLocalStorageString);
```
Comment (annix8) [inline tests/e2e/app-specs/verify-extensions.e2e-spec.ts:75]: we have a browser-helper.ts which holds many 1 line functions that execute browser functions. Maybe export a function from there and use it here just for consistency?

File: tests/e2e/app-specs/verify-extensions.e2e-spec.ts [hunk 2]
```diff
@@ -56,6 +62,25 @@ const ENTITY_MAP = new Map<string, any>()
...
+
+        browser.waitForAngularEnabled(false)
```
Comment (annix8) [inline tests/e2e/app-specs/verify-extensions.e2e-spec.ts:77]: same same

Comment (annix8) [inline tests/e2e/app-specs/verify-extensions.e2e-spec.ts:77]: same

File: tests/e2e/app-specs/verify-extensions.e2e-spec.ts (hunk 1 above)
Comment (Novkirishki) [reply tests/e2e/app-specs/verify-extensions.e2e-spec.ts]: done

File: tests/e2e/app-specs/verify-extensions.e2e-spec.ts (hunk 2 above)
Comment (Novkirishki) [reply tests/e2e/app-specs/verify-extensions.e2e-spec.ts]: done

**Review by annix8 — APPROVED** [review]

**Pull request closed without merging**